packages/core/test/util/array.test.ts

```diff
@@ -0,0 +1,26 @@
+import { moveUp, moveDown } from '../../src/util/';
+import test from 'ava';
+let array: Array<number> = [];
+test.beforeEach(() => {
+  array = [1, 2, 3, 4, 5];
```

Comment:
Haven't used ava before. Don't know if reassigning a common variable is fine or it will lead to race conditions. Please suggest here.